client/platform/desktop/backend/serializers/viame.spec.ts

```diff
@@ -3,7 +3,14 @@ import { MultiTrackRecord } from 'dive-common/apispec';
 import fs from 'fs-extra';
 import mockfs from 'mock-fs';
 import { JsonMeta } from 'platform/desktop/constants';
-import { serialize } from 'platform/desktop/backend/serializers/viame';
+import { serialize, parse } from 'platform/desktop/backend/serializers/viame';
+import { Attribute } from 'vue-media-annotator/use/useAttributes';
+import processTrackAttributes from 'platform/desktop/backend/native/attributeProcessor';
+
+
+type testPairs = [string[], MultiTrackRecord, Record<string, Attribute>];
+
+const testData: testPairs[] = fs.readJSONSync('../testutils/viame.spec.json');
```

Comment:
doesn't yarn always crawl back up to the package.json directory?  To do it from a subdirectly without a package.json wouldn't you have to do `yarn --cwd ./` (which would fail because it doesn't have package.json or a vue-cli-service)  I assume that if you were to install jest on npm globally and do `jest ./` from inside of platform it would fail (maybe because of missing config options) and from the root dir it would pass?

Reply:
I see, yarn sets the CWD before passing off execution to the tests.  Carry on then.